docs/full/policy.md

```diff
@@ -0,0 +1,132 @@
+
+## Plugin/Tanzu-CLI Compatibility policy
+
+- Each Tanzu Core CLI release should be compatible with all the existing plugins at the time of release and also with plugins developed with any future patch of the existing minor releases of Tanzu Plugin Runtime.
+  - For example, the below table shows the compatible plugin version with different releases of Tanzu CLI.
+  - Note: In the below table v0.X.Y versions are `legacy` library versions from Tanzu-Framework
+
+- Important: Based on this compatibility policy, the User can always upgrade the Tanzu CLI to the latest version without worrying about existing plugin compatibility. The latest minor or patch release of  Tanzu CLI under the v1 major release will always be compatible with all existing plugins.
+- This means any change in the contract between Tanzu CLI and Tanzu Plugin Runtime must be done in a backward-compatible manner.
+- FAQ:
+  - What happens if I upgrade to a newer version of CLI v1.2.0 and use a plugin developed with Plugin Runtime v1.0.5?
+    - As CLI is always compatible with all existing plugins, plugins developed with Plugin Runtime v1.0.5 should continue to work.
+  - If I have developed a plugin with pre-v1.0 (v0.11, 0.25, 0.28) of Tanzu Plugin Runtime (from Tanzu-Framework repository), Will my plugins be compatible with v1.0 Tanzu CLI?
+    - Yes. As shown in the above diagram and the backward compatibility guarantees provided with the v1.0 release of Tanzu CLI, all plugins developed with pre-v1.0 of Tanzu Plugin Runtime will be compatible with v1.0 Tanzu CLI.
+  - As a pre-v1.0 Tanzu CLI plugin developer, What are the possible changes I need to do when consuming v1.0 of Tanzu Plugin Runtime?
+    - We are still working on the design and the possible changes required when consuming v1.0 of Tanzu Plugin Runtime but high-level changes are mentioned in the Impact on Plugin Developers section below.
+  - When will my plugin be incompatible with the CLI?
+    - The user may be using a CLI version that is old and some new functionality implemented in a plugin with newer Plugin Runtime is not compatible with the old CLI. However, the old CLI will still be able to invoke the plugin with all the compatible features. Also, upgrading CLI to a newer version should resolve this new feature incompatibility issue.
+    - Example: If a plugin is developed with v1.3.1 Tanzu Plugin Runtime which introduced a new feature X which works in combination with a newer version of Tanzu CLI then if the user is using old v1.2.0 Tanzu CLI (which doesn’t support this new feature X  introduced in Plugin Runtime v1.3.1) while invoking a plugin, that new functionality might not work when using old CLI. However, other functionalities of that plugin will continue to work as expected with the old CLI. This will make the new feature in the plugin incompatible with the installed CLI. However, the user can upgrade the CLI to the latest v1.3.0 version to make the new feature compatible again.
```

Comment:
nit:
v1.3.1 Tanzu Plugin Runtime ->
Tanzu Plugin Runtime v1.3.1  or 
v1.3.1 of  Tanzu Plugin Runtime